src/containers/homepage/survey-list.component.tsx

```diff
@@ -30,20 +59,19 @@ export class SurveyListComponent extends Component<SurveyListComponentProps> {
                 title: 'Created at',
                 dataIndex: 'createdAt',
                 key: 'createdAt',
-                // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString
-                render: (date: number) => new Date(date * 1000).toLocaleDateString(),
+                render: (date: number) => new Date(date * 1000).toLocaleString(),
             },
             {
                 title: 'Last modified at',
                 dataIndex: 'modifiedAt',
                 key: 'modifiedAt',
-                render: (date: number) => new Date(date * 1000).toLocaleDateString(),
+                render: (date: number) => new Date(date * 1000).toLocaleString(),
```

Comment:
We should extract this into its own helper function. This way we can easily add a library (like Moment) later. The same code is also needed on the detail page.

Reply:
+1 on this

Reply:
:+1: